uisdk/src/adyen/res/values/styles.xml

```diff
@@ -426,7 +426,7 @@
         <item name="backgroundColor">@drawable/uisdk_booking_button_background</item>
         <item name="backgroundTint">@null</item>
         <item name="android:text">@string/kh_uisdk_book_now</item>
-        <item name="android:textColor">@color/kh_uisdk_text_white</item>
+        <item name="android:textColor">@color/kh_uisdk_background_primary</item>
```

Comment:
Same with using a background color for text. Also, is this background color from the new palette? Afaik they've all been named background 1, 2, 3, etc, not primary, secondary.